File: apps/planner/src/steps.ts
export const STEPS = [
  { id: 'welcome', label: 'Başla', num: 1 },
  { id: 'journey', label: 'Rota', num: 2 },
  { id: 'explore', label: 'Keşfet', num: 3 },
  { id: 'title', label: 'Başlık', num: 4 },
  { id: 'hotels', label: 'Konaklama', num: 5 },
  { id: 'food', label: 'Yemek', num: 6 },
  { id: 'plan', label: 'Plan', num: 7 },
  { id: 'calendar', label: 'Takvim', num: 8 },
  { id: 'publish', label: 'Yayın', num: 9 },
] as const;

export type StepId = (typeof STEPS)[number]['id'];

File: mobile/lib/features/planner/steps.dart
// apps/planner/src/steps.ts birebir karşılığı.

enum StepId { welcome, journey, explore, title, hotels, food, plan, publish }

class StepDef {
  const StepDef(this.id, this.label, this.num);
  final StepId id;
  final String label;
  final int num;
}

/// steps.ts STEPS dizisi — sıra ve etiketler birebir (i18n nav.* Türkçesi).
const List<StepDef> kSteps = [
  StepDef(StepId.welcome, 'Başla', 1),
  StepDef(StepId.journey, 'Rota', 2),
  StepDef(StepId.explore, 'Keşfet', 3),
  StepDef(StepId.title, 'Başlık', 4),
  StepDef(StepId.hotels, 'Konaklama', 5),
  StepDef(StepId.food, 'Yemek', 6),
  StepDef(StepId.plan, 'Plan', 7),
  StepDef(StepId.publish, 'Yayın', 8),
];

int stepIndex(StepId id) => kSteps.indexWhere((s) => s.id == id);
